telegram/ext/_utils/webhookhandler.py

```diff
@@ -146,6 +146,9 @@ async def post(self) -> None:
                 "Received data was *not* processed!",
                 exc_info=exc,
             )
+            raise tornado.web.HTTPError(
+                HTTPStatus.BAD_REQUEST, reason="Update could not be processed"
+            ) from None
```

Comment:
why `from None` and not `from exc`?

Reply:
hm, thought it wouldn´t make a difference to the end user but you are right its better with.